active/website.md

```diff
@@ -0,0 +1,63 @@
+# Website Working Group
+
+## Scope of responsibilities
+
+This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.
+
+The duties of the working group are:
+- Introduce new features on the website
+- Maintain and monitor the website 
+- Help to make the website accessible to all
+
+
+### Delegated responsibilities:
+- Members propose, triage, analize, discuss and implement new features
+- Members triage the project’s issues and pull requests.
+- Member maintain and monitor the website including updating versions.
+- Mentor new contributors to the website.
+- Run or support djangoproject.com sessions in DjangoCon sprints.
+- Chair, Co-Chair and Board Liaison can sign off on new features.
+
+## Initial membership
+
+- Chair: Sarah Abderemane
+- Co-Chair: Saptak Sengupta
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): Sarah Abderemane
+- Other members:
+    - Eric Sherman
+    - Mark Walker
+    - Jason Judkins
+    - Paolo Melchiorre
+    - Sanyam Khurana
+    - Tobias McNulty
+    - Ron Maravanyika
+    - alexgmin Alex
```

Comment:
Sorry didn't realize my github account didn't have my surname. Please change it to Alex Gómez.